src/deprecated.jl

```diff
@@ -130,3 +130,11 @@ function MultiSiteOperator(args...)
     )
     return multisite_operator(args...)
 end
+
+export ProgressBar
+function ProgressBar(args...)
```

Comment:
The origin `ProgressBar` has keyword arguments, maybe the following is better

```suggestion
function ProgressBar(args...; kwargs...)
```